bot/tests/mocks/phabricator_diff_search.json

```diff
@@ -1,33 +1,47 @@
 {
     "result": {
-        "data": [
```

Comment:
Please use `python -m json.tool bot/tests/mocks/phabricator_diff_search.json ` to format the file and avoid these diffs